### DISCUSSION TITLE
"Strict hold" mod for mania

### DISCUSSION THREAD

**Discussion by Bruno5430** [ppy/osu#24286] [Ideas]
Similar to strict tracking mod but for mania ruleset
This mod when applied would instantly break combo when releasing hold notes incorrectly, just like stable behaves

**Comment by peppy** [comment]
If it's how stable behaves, it should either be implemented as the default behaviour, or implemented via classic mod.

**Reply by Bruno5430** [reply]
I'm ok with the current behavior, but one of my friends disliked it when i explained how it works
So this being inside #23157 would be fine?
I didn't want to ask for it being implemented in classic mod because there is no classic mod implemented in mania currently
